src/app/src/main/java/br/com/italoromeiro/arctouchtest/utils/FormatterUtil.java

```diff
@@ -0,0 +1,15 @@
+package br.com.italoromeiro.arctouchtest.utils;
+
+/**
+ * Created by italo on 09/08/16.
+ */
+public class FormatterUtil {
```

Comment:
Quick question about naming: why is `FormatterUtil` in singular form? I'm asking this because the other two utilities classes are in the plural (`AlertUtils` and `PermissionUtils`).